routes/messages.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const router  = express.Router();
+
+module.exports = (db) => {
+
+  // show all the incoming messages
+  router.get("/", (req, res) => {
+
+   let queryString = `
+   SELECT messages.id, users.id AS sender_id, users.name AS sender_name, users.email, users.phone, items.title AS inquiry_about, messages.created_at, messages.body
+   FROM messages
+   JOIN users on sender_id = users.id
+   JOIN items on item_id = items.id
+   WHERE receiver_id = $1
+   ORDER BY sender_name, created_at ASC;
+    `;
+
+    db.query(queryString,[req.session.user_id])
+      .then(data => {
+        const messages = data.rows;
+        console.log(messages);
```

Comment:
TODO: remove these old console.log statements once you're done using them for debugging. :)